spinel-cli.py

```diff
@@ -1980,8 +1994,15 @@ def do_mac(self, line):
                 prop = SPINEL.PROP_MAC_MAX_RETRY_NUMBER_DIRECT
             elif params[1] == "indirect":
                 prop = SPINEL.PROP_MAC_MAX_RETRY_NUMBER_INDIRECT
-
-        value = params[2] if len(params) == 3 else None
+            value = params[2] if len(params) == 3 else None
+
+        elif params[0] == "ccathreshold" and len(params) > 0:
+            prop = SPINEL.PROP_PHY_CCA_THRESHOLD
+            value = None
+            if len(params) == 2:
+                value = int(params[1])
+                self.prop_set(prop, value, mixed_format='b')
```

Comment:
nit: What about to define variable `mixed_format` and use `self.handle_property` from below with additional parameter?

Reply:
I've tried this, but there is some unknown bug in handle_property and it's not possible to set value to "0" (all other values can be set properly).
I need to debug and fix handle_property first and then we can switch to use this function. For now I'd prefer to merge it as is to have support for this command in pyspinel.